src/many-identity-hsm/src/lib.rs

```diff
@@ -308,12 +308,72 @@ impl Hsm {
         }
         Ok(())
     }
+}
 
-    /// Default constructor. You should not call this and use HSM_INSTANCE instead
-    fn new() -> Hsm {
-        Hsm {
-            ..Default::default()
-        }
+#[derive(Clone)]
+pub struct HsmIdentity {
```

Comment:
You removed the ability to perform hardware verification of messages. Isn't hardware offloading something we wanted for, e.g., servers?

Reply:
Right. Let me see if I can re-add it.

Reply:
I don't see this resolved, but I won't block the merge. You can create a new issue and add it to another PR.

Reply:
This isn't used anywhere yet. We should file an issue and prioritize it properly.

Reply:
https://github.com/liftedinit/many-rs/issues/190